install-raccine.bat

```diff
@@ -86,6 +86,7 @@ IF '%errorlevel%' NEQ '0' (
 REGEDIT.EXE /S raccine-reg-patch-wmic.reg 
 REGEDIT.EXE /S raccine-reg-patch-wbadmin.reg
 REGEDIT.EXE /S raccine-reg-patch-bcdedit.reg
+REGEDIT.EXE /S raccine-reg-patch-pshell.reg
```

Comment:
Please rename the file to raccine-reg-patch-powershell.reg

Reply:
Done